src/types/transcript-item-data.ts

```diff
@@ -0,0 +1,44 @@
+export type HighlightedMap = Record<string, boolean>;
+
+export interface RawItemData {
+  cuePointType: ItemTypes;
+  text?: string;
+  description?: string;
+  title?: string;
+  assetId?: string;
+  subType?: ItemTypes;
+  partnerData?: string;
+  tags?: string;
+  assetUrl?: string;
+  relatedObjects?: {
+    QandA_ResponseProfile?: {
+      objects: Array<{xml: string}>;
+    };
+  };
+}
```

Comment:
The interface incorrect. Raw data has next properties:
<img width="515" alt="image" src="https://user-images.githubusercontent.com/51074448/181211100-1bf7692d-1fa7-4b32-b702-cbe871b645b1.png">